project_script/messages.py

```diff
@@ -0,0 +1,2 @@
+def message_invalid_path():
+    return print("Arquivo inválido ou inexistente \n")
```

Comment:
As like the most  languages needed the blank line at the end of the file, to avoid possibles conflicts, the github itself hint to do this.

![image](https://user-images.githubusercontent.com/11571434/159491671-6a5274fb-25b8-415d-8c97-6e4c5b15b7c5.png)